### ISSUE TITLE
works on first load but then I get "cannot ready property 'desktop' of undefined

### ISSUE THREAD

**Issue by vesper8** [jonhue/turbolinks-animate.js#4]
I added the demo code exactly as is

```
$(document).on( 'turbolinks:load', function() {
    $('body').turbolinksAnimate();
    turbolinksAnimateAppear();
});
$(document).on( 'turbolinks:request-start', function() {
    turbolinksAnimateDisappear();
});
$(window).on( 'popstate beforeunload', function(event) {
    turbolinksAnimateDisappear();
});
```

It works beautifully on initial load, but then on subsequent clicks I get this error

<img width="687" alt="screenshot 2017-12-27 19 09 49" src="https://user-images.githubusercontent.com/816028/34396486-8df8ff02-eb39-11e7-8be4-cf241ac9665c.png">

any idea what could be wrong?

**Comment by vesper8** [comment]
I've isolated the problem to the

```
$(document).on('turbolinks:request-start', function () {
    turbolinksAnimateDisappear();
});
```

And the problem goes away (and animations become smooth and beautiful if I add

```
$(document).on('turbolinks:request-start', function () {
    $('body').turbolinksAnimate();
    turbolinksAnimateDisappear();
});
```

Are there consequence to doing this?

**Comment by jonhue** [comment]
Could you check if adding `'fadein'` as an inline animation to your `body` element resolves the problem?

```html
<body data-turbolinks-animate-animation="fadein">
```

**Comment by shrmnk** [comment]
I am experiencing the exact same issue.

@jonhue Adding that linein animation resolves the issue.

**Comment by jonhue** [comment]
Ah..

The bug is pretty easy to find just reading the code :-)
https://github.com/jonhue/turbolinks-animate.js/blob/1.3.4/turbolinks-animate.js#L93

I think I might have assumed that `undefined` would work just as `null` does.

**Comment by jonhue** [comment]
fixed in 1.3.5
